.github/workflows/release-sdk-to-candidate.yml

```diff
@@ -0,0 +1,50 @@
+name: Release ffmpeg-2204-sdk to candidate
+
+on:
+  # Run the workflow each time new commits are pushed to the candidate branch. 
+  push:
+    branches: [ "2204" ]
+    paths: 
+      - "ffmpeg-2204-sdk/**"
+  workflow_dispatch:
+    
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  get-architectures:
+    name: 🖥 Get snap architectures
+    runs-on: ubuntu-latest
+    outputs:
+      architectures: ${{ steps.get-architectures.outputs.architectures }}
+      architectures-list: ${{ steps.get-architectures.outputs.architectures-list }}
+    steps:
+      - name: 🖥 Get snap architectures
+        id: get-architectures
+        uses: snapcrafters/ci/get-architectures@main
+        with:
+          snapcraft-project-root: "ffmpeg-2204-sdk"
+
+  release-sdk:
+    name: 🚢 Release to latest/candidate
+    needs: get-architectures
+    runs-on: ubuntu-latest
+    environment: "2204 Branch"
+    strategy:
+      matrix:
+        architecture: ${{ fromJSON(needs.get-architectures.outputs.architectures-list) }}
+    steps:
+      - name: 🚢 Release to latest/candidate
+        uses: snapcrafters/ci/release-to-candidate@main
+        with:
+          architecture: ${{ matrix.architecture }}
+          channel: "candidate"
+          launchpad-token: ${{ secrets.LP_BUILD_SECRET }}
+          snapcraft-project-root: "ffmpeg-2204-sdk"
+          store-token: ${{ secrets.SNAP_STORE_CANDIDATE }}
```

Comment:
This file is missing the call for testing phase

Reply:
There is nothing to test in a sdk. The content snap is only what we can test.

Reply:
Right, but we still need a control mechanism for getting newly released revisions from `candidate` -> `stable`.

Reply:
Uhh.. we can't directly push it right? Hmm.. is it possible, to promote both to stable together?